bundles/io.openliberty.tools.eclipse.ui/src/io/openliberty/tools/eclipse/DebugModeHandler.java

```diff
@@ -182,22 +206,21 @@ protected IStatus run(IProgressMonitor monitor) {
                         return Status.CANCEL_STATUS;
                     }
 
-                    String portToConnect = waitForSocketActivation(project, DEFAULT_ATTACH_HOST, debugPort, monitor);
+                    String portToConnect = waitForSocketActivation(project, DEFAULT_ATTACH_HOST, debugPort, projDebugPort, monitor);
```

Comment:
In the case that the server has already been started, this is just going to return a 'null', and so we'll go on to return the OK_STATUS rather than the ERROR.  This switches to the debug perspective possibly making it harder to see the terminal with the error now.